README.md

```
@@ -17,16 +17,27 @@ https://alexpung.github.io/UK-Investment-tax-calculator/
 ## Supported import format and brokers
-| Category                | IB XML | FreeTrade CSV        | Trading212 CSV       |
-|-------------------------|--------|----------------------|-----------------------|
-| CashSettlement (Option) | O      |                      |                       |
-| StockSplit              | O      |                      |                       |
-| Dividend                | O      | Total in GBP         | Total in GBP          |
-| FutureContractTrade     | O      |                      |                       |
-| FxTrade                 | O      |                      |                       |
-| InterestIncome          | O      | Saving income in GBP | Saving income in GBP  |
-| OptionTrade             | O      |                      |                       |
-| Trade (Stock)           | O      | Total in GBP         | Total in GBP          |
+| Category                | IB XML | FreeTrade CSV        | Trading212 CSV       | interactive investor CSV (experimental) |
```

markdownlint-cli2 (0.23.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)